core/storage/topic/gae_models.py

```diff
@@ -249,7 +249,8 @@ def get_model_association_to_user():
 
     @classmethod
     def get_export_policy(cls):
-        """This model is only stored for archive purposes. The commit log of
+        """Model doesn't contain any data directly corresponding to a user.
```

Comment:
Here & for the other commit log entry models in this PR, the wording seems a bit off. These models DO contain data corresponding to a user, but it doesn't correspond to personal user data, it's not useful for users in the context of takeout, and it's used by us for archive purposes.

Please clarify the wording here & for all other commit log entry models.

Reply:
Done